.github/PULL_REQUEST_TEMPLATE/leaf_package_update.md

```diff
@@ -0,0 +1,65 @@
+<!--
+This project's monthly turnover might well be beyond 1500 PRs. Since we are community driven,
+let's do tit for tat: submitters are kindly asked to pick an open pull request for review.
+
+Start with packages updates. They are relatively simple.
+Since you'll be required to do those very steps below, it'll be easy.
+
+Marvin needs_reviewer: https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+is%3Aopen+label%3Aneeds_reviewer+
+Marvin needs_merger:   https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+is%3Aopen+label%3Aneeds_merger+
+Reviewing guidelines:  https://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/download/1/nixpkgs/manual.html#chap-reviewing-contributions
+-->
+
+##### Motivation for this _leaf_ / _near leaf_ package update
+<!-- 
+"leaf" or "near leaf" packages have a relatively low (or zero) number of packages that
+depend on it. The below workflow only performs sufficiently on such packages.
+
+For "core" or "near core" packages, use the alternative PR template.
+-->
+
+
+##### General review
+
+- [ ] package name fits guidelines
+- [ ] package version fits guidelines
+- [ ] fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).
+
+##### Package build
+<!-- Tip: pipe those commands through bash directly from your editor -->
+
+```
+uname -srm
+---
+nix-shell -p lsb-release --run "lsb_release -a" 2> /dev/null
+```
+
+<details>
+<summary>Logs from <code>nixpkgs-review rev HEAD</code></summary>
+
+```
+nixpkgs-review rev HEAD
+
+```
+</details>
+
+<!--
+substitute this command by it's own output when run within the nipkgs-review shell
+-->
+cat ./report.md
+
+<!--
+substitute this command by it's own output when run within the nipkgs-review shell
+and test "not ok" binaries manually
+-->
+for cmd in $(ls ./results/**/**/**); do if $($cmd --help > /dev/null); then echo "- [x] \`$cmd --help\`: ok"; else "- [ ] \`$cmd --help\`: not ok -- tested otherwise"; fi; done
```

Comment:
@Mic92 Let's replace this whole section by standardized output from the `nixpkgs-review`'s `report.md`

- We could add some logs, so any pull requester can get efficient hands on help directly by an experienced reviewer if they hit an obstacle.
- https://github.com/Mic92/nixpkgs-review/pull/117 is already in the making